baram/ec2_manager.py

```diff
@@ -1,68 +1,217 @@
 import boto3
+import botocore.exceptions
+
+from baram.log_manager import LogManager
+from baram.sagemaker_manager import SagemakerManager
```

Comment:
@Baescott 

IMO, you don't need to import SM and making getting sm redundant SG.

I'd rather make more general method such as `get_redundant_sg`.

Reply:
Thanks for giving sustainable opinion, I'll reflect it asap